PyFlyt/core/abstractions.py

```diff
@@ -25,6 +25,7 @@ def __init__(
         p: bullet_client.BulletClient,
         start_pos: np.ndarray,
         start_orn: np.ndarray,
+        start_vel: np.ndarray,
```

Comment:
Start velocity defined here but not used later in loadURDF. If this is important to be here, also update the `quadx.py` file to have this, but if this is not needed, find a way to put it into `fixedwing.py`, it can be placed under the reset.

For now, hardcode it into the `fixedwing.py` code, I will configure aviary to have vehicle specific kwargs later.

Reply:
As far as I can tell, its not possible to set initial velocity in loadURDF, atm its passed as a kwarg for `fixedwing.py` and used in `self.p.resetBaseVelocity` in reset. 

Will remove `start_vel` and hardcode it for now